app/routes/auth/contract.js

```diff
@@ -16,7 +16,7 @@ export default Ember.Route.extend({
     return this.get('store').queryRecord('developer',
       {
         username: params.username,
-        token: '~'
+        token: '!'
```

Comment:
for magic values like this, it would be helpful to designate a constant for them. In other words:
```
const HAS_NOT_ACCEPTED_CONTRACT = '!'
const TOKEN_REVOKED = '~'
...
{
  username: params.username,
  token: HAS_NOT_ACCEPTED_CONTRACT
}
```
Would be much more readable and maintainable.